src/attributes.js

```diff
@@ -70,6 +70,22 @@ const applyProp = function(el, name, value) {
 
 
 /**
+ * Applies a value to a style declaration. Supports CSS custom properties by
+ * setting properties containing a dash using CSSStyleDeclaration.setProperty.
+ * @param {CSSStyleDeclaration} style
+ * @param {!string} prop
+ * @param {*} value
+ */ 
+const setStyleValue = function(style, prop, value) {
+  if (prop.indexOf('-') >= 0) {
+    style.setProperty(prop, /** @type {string} */(value));
```

Comment:
I think that'll be the same performance. (always `#setProperty` is faster than testing than using, but not faster than just using the property). This LGTM.